packages/zowe-explorer/__tests__/__unit__/dataset/DatasetTree.unit.test.ts

```diff
@@ -2700,6 +2700,8 @@ describe("Dataset Tree Unit Tests - Function initializeFavorites", () => {
     });
 });
 describe("Dataset Tree Unit Tests - Sorting and Filtering operations", () => {
+    createGlobalMocks();
+    mocked(vscode.window.createTreeView).mockReturnValueOnce(createTreeView());
```

Comment:
something to consider for the future: I had to change a bunch of mock references so that `vscode.window.createTreeView` didn't break, we might want a central resource for VS code mocks so that we do not have to update them in several places. I can see the possibility for this happening again if we would leverage another `TreeView` function that is not being returned from these mocks.